src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -467,8 +467,6 @@ internal Arm64() { }
         ///   <para>svuint8_t svand[_u8]_x(svbool_t pg, svuint8_t op1, svuint8_t op2)</para>
         ///   <para>svuint8_t svand[_u8]_z(svbool_t pg, svuint8_t op1, svuint8_t op2)</para>
         ///   <para>  AND Zresult.D, Zop1.D, Zop2.D</para>
-        ///   <para>svbool_t svand[_b]_z(svbool_t pg, svbool_t op1, svbool_t op2)</para>
-        ///   <para>  AND Presult.B, Pg/Z, Pop1.B, Pop2.B</para>
```

Comment:
Why remove these? Aren't we going to be emitting them long term?

The logic is mostly all there, just enabling it for SVE didn't happen in .NET 9

Reply:
Right, we wanted to keep only the instructions in the summary docs that are enabled in .NET 9.

Reply:
Its all experimental anyways, so does it matter?

Seems like extra churn and backporting that isn't strictly needed. We didn't do that for AVX512 for example

Reply:
alright, will skip this one.